src/chapters/typescript/functions.rst

```diff
@@ -0,0 +1,115 @@
+Functions in TypeScript
+=======================
+
+When creating functions in TypeScript, we have many of the same options as in JavaScript.
+We can make anonymous functions, give it a different number of parameters and so on.
+However, when working with functions in TypeScript, we have to keep *type* in mind.
+
+Declaring Functions
+-------------------
+
+Named Functions
+^^^^^^^^^^^^^^^
+
+Let's take a look at a TypeScript function declaration.
+
+.. sourcecode:: typescript
+
+   function myFunction(a: number): number {
+      return a*2;
+   }
+
+Here you can see that we provided the type of the parameters and the type of the value that is returned.
+If the function doesn't have a return value, then you use ``void`` for its type.
+
+.. note::
+
+   While in many cases, it is possible to determine the return type during compilation, if you know the return type, you should give it. 
```

Comment:
also an example to showcase the syntax if you don't know the return type?

Reply:
Removed this note. Reread some other sources and it seems like it is not worth covering.